go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/ipfs/go-datastore v0.5.1
 	github.com/ipfs/go-detect-race v0.0.1
 	github.com/ipfs/go-ds-badger v0.3.0
-	github.com/ipfs/go-ds-flatfs v0.5.1
+	github.com/ipfs/go-ds-flatfs v0.5.0
```

Comment:
Was this intentional? Do we want to downgrade this?

Reply:
No, it was introduced with the very first commit, possible regression due to rebase.
Pushed commit that reverts this and other downgrades like this.